test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetEndToEndTests/NuGetConsoleTestCase.cs

```diff
@@ -291,6 +292,226 @@ public async Task InstallAndUpdatePackageWithSourceParameterWarnsAsync(ProjectTe
             }
         }
 
+        [NuGetWpfTheory]
+        [MemberData(nameof(GetPackagesConfigTemplates))]
+        public async Task InstallPackageFromPMCForPC_PackageNamespace_WithSingleFeed(ProjectTemplate projectTemplate)
```

Comment:
nit: PMC is not super relevant, the test class has only PMC tests :)

Reply:
Sure, removed.